velox/docs/functions/spark/string.rst

```diff
@@ -141,6 +141,18 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
         SELECT overlay('Spark SQL', 'tructured', 2, 4); -- "Structured SQL"
         SELECT overlay('Spark SQL', '_', -6, 3); -- "_Sql"
 
+.. spark:function:: repeat(str, n) -> varchar
+
+    Returns the string which repeats the given string value ``n`` times. 
+    ``n`` must be less or equal than 10000.
+    If ``n`` is less or equal than 0, empry string is returned. ::
+
+        SELECT repeat('123', 2); -- 123123
+
+.. spark:function:: string_repeat(str, n) -> varchar
```

Comment:
Please also put this in alphabetical order, though it's just for documenting alias.